Motion Amendments 2019 - Senior Delegates (#27)
thewca/wca-documents: documents/motions/09.2017.1 - Senior Delegates.md

      2. Perform as the escalation point for the WCA Delegates in case of issues with local Competition Organizers and Competition Staff.
4. Appointment of Senior Delegates
   1. The Board of Directors decides how many Senior Delegates are needed and how the areas are divided among Senior Delegates, based on the number of delegates and the span-of-control of the Senior Delegates. All WCA Delegates should be managed by a Senior Delegate.

>    2. If a new Senior Delegate is needed for an area, the Board of Directors will request all Full Delegates to apply as a candidate.

Reviewer: This has always sounded weird to me, so while we are at it, why don't we reword it a bit?
```suggestion
   2. If a new Senior Delegate is needed for an area, the Board of Directors will ask for applicants among all Full Delegates.
```
